todo_app/app.py

```diff
@@ -1,47 +1,92 @@
 import os
+import logging
+import logging.config
+import loggly.handlers
+import time
+from dotenv import load_dotenv
 from flask import Flask, render_template, request, redirect, url_for
 from todo_app.flask_config import Config
 from todo_app.data.view_model import ViewModel
 from todo_app.data.items import ItemService
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
+
+logging.config.fileConfig('python.conf')
+logging.Formatter.converter = time.gmtime
+logger = logging.getLogger('myLogger')
+
+logger.info('Test log')
+
 
 def create_app():
     app = Flask(__name__)
+    load_dotenv()
```

Comment:
This shouldn't be necessary - in Docker we load the environment variables into the container, and locally this should be covered by the `python-dotenv` library which will automatically get called when Flask starts